app/models/form/lettings/questions/created_by_id.rb

```diff
@@ -8,21 +10,27 @@ def initialize(id, hsh, page)
   end
 
   def answer_options
-    answer_opts = { "" => "Select an option" }
-    return answer_opts unless ActiveRecord::Base.connected?
+    return ANSWER_OPTS unless ActiveRecord::Base.connected?
```

Comment:
if we are touching any `ActiveRecord::Base.connected?` can we start removing them? i don't understand why we code for this, if we can't connect to the DB we won't know who the user is any way

Reply:
i don't see why not, if we can't connect to the database you won't even be able to authenticate the user